### ISSUE TITLE
Error when used 'detected_objects' in Android

### ISSUE THREAD

**Issue by AnMokoto** [google/mediapipe#2161]
Error when used 'detected_objects' in Android

i'd build an `aar` below:
```bazel
load("//mediapipe/java/com/google/mediapipe:mediapipe_aar.bzl", "mediapipe_aar")

mediapipe_aar(
    name = "android_mediapipe",
    calculators = ["//mediapipe/graphs/object_detection_3d:mobile_calculators"],
)

-------------------------------------------------------

bazel build --strip=always -s -c opt  \
--host_crosstool_top=@bazel_tools//tools/cpp:toolchain \
 --fat_apk_cpu=arm64-v8a,armeabi-v7a \
//mediapipe/examples/android/src/java/com/google/mediapipe/apps/aar:android_mediapipe.aar
```

then build the `binarypb`:
```shell
bazel build -c opt //mediapipe/graphs/object_detection_3d:mobile_gpu_binary_graph
```

WHEN I USED [FrameProcessor#addPacketCallback(String outputStream, PacketCallback callback)](https://github.com/google/mediapipe/blob/a9b643e0f5978425d948050a2228398929733d05/mediapipe/java/com/google/mediapipe/components/FrameProcessor.java#L333) WITH 'STREAM NAME' SAME AS  ['detected_objects'](https://github.com/google/mediapipe/blob/a9b643e0f5978425d948050a2228398929733d05/mediapipe/python/solutions/objectron.py#L256), I GOT SOME ERROR MESSAGE:
```kotlin
 processor?.addPacketCallback("detected_objects"){
                   //...
}
```

```java
com.google.mediapipe.framework.MediaPipeException: unknown: ; Input Stream "detected_objects" for node with sorted index 2 does not have a corresponding output stream.
        at com.google.mediapipe.framework.Graph.nativeStartRunningGraph(Native Method)
        at com.google.mediapipe.framework.Graph.startRunningGraph(Graph.java:324)
        at com.google.mediapipe.components.FrameProcessor.startGraph(FrameProcessor.java:553)
        at com.google.mediapipe.components.FrameProcessor.maybeAcceptNewFrame(FrameProcessor.java:421)
        at com.google.mediapipe.components.FrameProcessor.onNewFrame(FrameProcessor.java:439)
        at com.google.mediapipe.components.ExternalTextureConverter$RenderThread.renderNext(ExternalTextureConverter.java:364)
        at com.google.mediapipe.components.ExternalTextureConverter$RenderThread.lambda$onFrameAvailable$0$ExternalTextureConverter$RenderThread(ExternalTextureConverter.java:309)
        at com.google.mediapipe.components.-$$Lambda$ExternalTextureConverter$RenderThread$IXJjtE68JMV2CMJNQcFYhTUqhj8.run(Unknown Source:4)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:224)
        at com.google.mediapipe.glutil.GlThread.run(GlThread.java:141)
```

**Thanks for your help How to get the data  about 'detected_objects' in `object_detection_3d`?**

**Comment by jiuqiant** [comment]
Please assign this to objectron owners. I think it's because the graph doesn't have an input stream called "detected_objects".

**Comment by AnMokoto** [comment]
thx for @jiuqiant  reply. i fixed the issue with #1893.
